eotsmanager/cmd/eotsd/daemon/unlock.go

```diff
@@ -5,27 +5,37 @@ import (
 	"fmt"
 	eotsclient "github.com/babylonlabs-io/finality-provider/eotsmanager/client"
 	"github.com/spf13/cobra"
+	"golang.org/x/term"
+	"os"
 )
 
+var UnlockCmdPasswordReader = passwordReader
+
+func passwordReader(cmd *cobra.Command) (string, error) {
+	cmd.Print("Enter password to unlock keyring: ")
+	passphrase, err := term.ReadPassword(int(os.Stdin.Fd()))
```

Comment:
given the unlock cmd can be executed remotely, will the remote operator get the prompt for password?